prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,72 @@
+"""
+Taxi Simulator
+"""
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    taxis = [
+        Taxi("Prius", 100),
+        SilverServiceTaxi("Lexus", 200, 2),
+        SilverServiceTaxi("Hummer", 150, 4)
+    ]
+    total_cost = 0.0
+    chosen_taxi = None
+
+    print("Let's drive!")
+    while True:
+        print("\nMenu: (q)uit, (c)hoose taxi, (d)rive")
+        choice = input(">>> ").lower()
+
+        if choice == "q":
+            break
+        elif choice == "c":
+            chosen_taxi = choose_taxi(taxis)
+        elif choice == "d":
+            if chosen_taxi:
```

Comment:
Consider refactoring low-level details (lines 27-30) into a function for consistency in level of detail in main() to improve main()'s readability